sandbox/tests/test scenes/lights/21 - non physical light shadow off.appleseed

```diff
@@ -0,0 +1,239 @@
+                <parameter name="enable_dl" value="true" />
+                <parameter name="enable_ibl" value="true" />
+                <parameter name="ibl_env_samples" value="1" />
+                <parameter name="next_event_estimation" value="true" />
+                <parameter name="optimize_for_lights_outside_volumes" value="True" />
+                <parameter name="record_light_paths" value="false" />
+                <parameter name="rr_min_path_length" value="6" />
+                <parameter name="volume_distance_samples" value="2" />
+            </parameters>
+            <parameters name="uniform_pixel_renderer">
+                <parameter name="decorrelate_pixels" value="true" />
+                <parameter name="force_antialiasing" value="true" />
+                <parameter name="samples" value="64" />
+            </parameters>
+        </configuration>
+        <configuration name="final" base="base_final">
+            <parameter name="lighting_engine" value="pt" />
+            <parameter name="passes" value="1" />
+            <parameter name="pixel_renderer" value="uniform" />
+            <parameter name="shading_result_framebuffer" value="ephemeral" />
+            <parameters name="adaptive_pixel_renderer">
+                <parameter name="max_samples" value="64" />
+                <parameter name="min_samples" value="16" />
+                <parameter name="quality" value="3.0" />
+            </parameters>
+            <parameters name="generic_frame_renderer">
+                <parameter name="tile_ordering" value="spiral" />
+            </parameters>
+            <parameters name="generic_tile_renderer">
+                <parameter name="max_samples" value="64" />
+                <parameter name="min_samples" value="16" />
+            </parameters>
+            <parameters name="light_sampler">
+                <parameter name="algorithm" value="cdf" />
+            </parameters>
+            <parameters name="pt">
+                <parameter name="dl_light_samples" value="1" />
+                <parameter name="enable_caustics" value="false" />
+                <parameter name="enable_dl" value="true" />
+                <parameter name="enable_ibl" value="true" />
+                <parameter name="ibl_env_samples" value="1" />
+                <parameter name="next_event_estimation" value="true" />
+                <parameter name="optimize_for_lights_outside_volumes" value="True" />
+                <parameter name="record_light_paths" value="false" />
+                <parameter name="rr_min_path_length" value="6" />
+                <parameter name="volume_distance_samples" value="2" />
+            </parameters>
+            <parameters name="uniform_pixel_renderer">
+                <parameter name="decorrelate_pixels" value="true" />
+                <parameter name="force_antialiasing" value="true" />
+                <parameter name="samples" value="64" />
```

Comment:
That's quite a high sampling rate for a test scene. What's the render time like?

We try to have test scenes render in seconds rather than minutes, because there are so many of them (and sometimes we render the full test suite with a debug build of appleseed, that takes 18-24 hours already).

Reply:
With a release build it takes 5.856 seconds. But that's with 16 threads. With 4 threads it's around 14.376 seconds. Should 16 samples be enough?

Reply:
Oh that's quite fast, it's fine with 64 spp then. Thanks!

Reply:
Ah I see that you changed it to 16 spp, that's fine too.